Ports/libslirp/package.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='libslirp'
+version='4.7.0'
+workdir="libslirp-v${version}"
+files="https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v${version}/libslirp-v${version}.tar.gz libslirp-v${version}.tar.xz 9398f0ec5a581d4e1cd6856b88ae83927e458d643788c3391a39e61b75db3d3b"
+auth_type='sha256'
+useconfigure='true'
+configopts=("--cross-file" "${SERENITY_BUILD_DIR}/meson-cross-file.txt")
+depends=("glib")
```

Comment:
Non-interpolated strings should use single quotes instead of double quotes.

Reply:
This doesn't look resolved?

Reply:
Ooops, missed this one. Fixed now